src/components/calendar/calendar.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import Calendar from './calendar';
+import { CalendarIcon, ExpandIcon } from '../icons';
+
+describe('<Calendar />', () => {
```

Comment:
@andela-akolawole please make sure to deep-test your components just like @abiodun0 has pointed out.